frontend/src/pages/manufacturing/create-order/CreateOrder.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState } from 'react';
+import { ReturnButton, Card, Progress } from '../../../components';
+import { Item } from '../../../interfaces/Items';
+import { Button, TextField, InputAdornment } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+
+import './CreateOrder.scss';
+import { useSnackbar } from '../../../contexts';
+import { useHistory } from 'react-router';
+
+interface Order {
+  good?: Item;
+  compositeId?: number;
+  quantity: number;
+}
+
+export function CreateOrder() {
```

Comment:
Either for this sprint or another, but there is a button to create an order for a specific item (clicking "order item" in the item-info route).
We should use /manufacturing/create-order/:id to display the item to order instead of just arriving at the default manufacturing page